LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Jonas Scheid, Steffen Lemke, Leon Bichmann, Marissa Dubbelaar
+Copyright (c) The nf-core/mhcquant team
```

Comment:
That sounds way nicer 😄

Reply:
Agreed :D. Still its good to know whom to ping if there are issues. nf-core migh use CODEOWNERS. So I would suggest to highlight it there as well